### PR TITLE
Implemented 3d Object Sheet wrapping either in Modeler and in Object3d

### PR DESCRIPTION
**Pull request by maxcapodi78**
#868

### CODE REVIEW

File: pyaedt/modeler/Object3d.py [hunk 1]
```diff
@@ -2695,6 +2695,34 @@ def subtract(self, tool_list, keep_originals=True):
...
+    def wrap_sheet(self, object_name, imprinted=True):
+        """Execute the sheet wrapping around an object. This object can be either the sheet or the object.
+        If wrapping produces an unclassified operation it will be reverted.
+
+        Parameters
+        ----------
+        object_name : str, :class:`pyaedt.modeler.Object3d.Object3d`
+            Object name or solid object or sheet name.
+        imprinted : bool, optional
+            Either if imprint or not over the sheet. Default is `False`.
```
Comment (MaxJPRey) [inline pyaedt/modeler/Object3d.py:2708]: @maxcapodi78 In the method signature, you set it to `True`. Maybe it should be `False` in the signature.
```suggestion
            Either if imprint or not over the sheet. Default is ``True``.
```

File: pyaedt/modeler/Modeler.py [hunk 2]
```diff
@@ -5090,3 +5090,45 @@ def flatten_assembly(self):
...
+        If wrapping produces an unclassified operation it will be reverted.
+
+        Parameters
+        ----------
+        sheet_name : str, :class:`pyaedt.modeler.Object3d.Object3d`
+            Sheet name or sheet object.
+        object_name : str, :class:`pyaedt.modeler.Object3d.Object3d`
+            Object name or solid object.
+        imprinted : bool, optional
+            Either if imprint or not over the sheet. Default is `False`.
```
Comment (MaxJPRey) [inline pyaedt/modeler/Modeler.py:5106]:
```suggestion
            Either if imprint or not over the sheet. Default is ``False``.
```

File: pyaedt/modeler/Object3d.py [hunk 3]
```diff
@@ -2695,6 +2695,34 @@ def subtract(self, tool_list, keep_originals=True):
         self._primitives.modeler.subtract(self.name, tool_list, keep_originals)
         return self
 
+    @pyaedt_function_handler()
+    def wrap_sheet(self, object_name, imprinted=True):
```
Comment (maxcapodi78) [inline pyaedt/modeler/Object3d.py:2699]:
```suggestion
    def wrap_sheet(self, object_name, imprinted=False):
```

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Pull request merged**